docs/en/install-upgrade/upgrading-elasticsearch.asciidoc

```diff
@@ -358,4 +358,4 @@ upgraded master.
 If you upgrade an {es} cluster that uses deprecated cluster or index settings
 that are not used in the target version, they are archived and ignored.
```

Comment:
This guidance got updated with https://github.com/elastic/elasticsearch/pull/82069.
```suggestion
that are not used in the target version, they are archived. We
recommend you remove any archived settings after upgrading.
```